ovos_PHAL_plugin_oauth/__init__.py

```diff
@@ -214,12 +221,22 @@ def handle_generate_qr(self, message):
 
         plugin_service_url = self.build_plugin_service_url(
             oauth_url, skill_id, app_id, client_id, client_secret)
+        qr_code = self.generate_qr(plugin_service_url, skill_id, app_id)
         self.bus.emit(message.reply("oauth.generate.qr.response", {
             "skill_id": skill_id,
             "app_id": app_id,
-            "qr": self.generate_qr(plugin_service_url, skill_id, app_id)
+            "qr": qr_code
         }))
 
+        # display the code in shell if registed app wants
+        display_code_on_shell = data.get("shell_integration")
```

Comment:
value should never be missing here i think, but should also default to True if somehow we get in this handler and the key is missing from data